BackfillPowerBI.yml

```diff
@@ -39,7 +39,7 @@ jobs:
   - pwsh: |
       try {
           (Get-ChildItem -Filter 'SimeonSyncPowerBI.psm1' -Recurse)[0].FullName | Import-Module -Force
-          Connect-PowerBIAsServicePrincipal -AppId $env:AppId -AppSecret $(Secret) -Tenant $env:Tenant
+          Connect-PowerBIAsServicePrincipal -AppId $env:PowerBIAppId -AppSecret $(PowerBIAppSecret) -Tenant $env:PowerBITenant
```

Comment:
how are these env variables getting populated? I would have expected them to all just look like $(PowerBIXyz)